schemas/senv_data.fbs

```diff
@@ -0,0 +1,21 @@
+//NOTE: This schema is under development.
+//If you decide to use it, be aware that backwards compatibility is not guaranteed.
+
+//Used to transmit fast sample environment data
+
+file_identifier "senv";
+
+enum Location : byte { Unknown = 0, Start, Middle, End }
+
+table senv_data {
+    Name: string (required);     // Name of the device plus channel number (e.g. "SomeName_3")
+    Channel: int;                // Channel number, currently a value from 0 to 3
+    PacketTimeStamp: ulong;      // The time stamp of the first sample in the value vector
+    TimeDelta: double;           // Time in nano seconds between samples
```

Comment:
Why double, is there a chance of requiring greater than nanosecond precision?

Reply:
Having nanosecond resolution is much better than required when only worrying about the time delta between two samples. However, if you want to use the delta to calculate the time stamp of sample number 1000, there is a possibility of being off by 500 ns in some circumstances.